tests/integration-tests/tests/api/test_api_client/test_custom_image/image.config.yaml

```diff
@@ -0,0 +1,10 @@
+Build:
+  InstanceType: {{ instance }}
+  ParentImage: {{ parent_image }}
+  UpdateOsAndReboot: true
+
+DevSettings:
+  Cookbook:
+    ChefCookbook: https://github.com/aws/aws-parallelcluster-cookbook/tarball/refs/heads/develop
+  NodePackage: https://github.com/aws/aws-parallelcluster-node/tarball/refs/heads/develop
+  AwsBatchCliPackage: https://github.com/aws/aws-parallelcluster/tarball/refs/heads/develop
```

Comment:
I believe you can remove these since they are injected by the framework when given as param to the test-runner